client/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
```

Comment:
We could have just one compose file at the root of the project

Reply:
Ultimately, we'll have just one docker-compose file when the backend project is up. We can delete this file when we revisit it in [CU-k0uuva](https://app.clickup.com/t/k0uuva)

Reply:
sounds good